scripts/completion/dvc.bash

```diff
@@ -10,33 +10,54 @@
 
 _dvc_commands='init destroy add import checkout run pull push fetch \
               status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit'
+              install root lock unlock pipeline commit cache cache_dir diff \
+              config index metrics_add metrics_modify metrics_remove move \
+              metrics_show pipeline_list pipeline_show remote_add version \
+              remote_list remote_default'
 
 _dvc_options="-h --help -v --version"
 _dvc_global_options="-h --help -q --quiet -V --verbose"
 
-_dvc_add="-R --recursive -f --file --no-commit"
-_dvc_checkout="$(compgen -G '*.dvc')"
+_dvc_add="-R --recursive -f --file --no-commit $(compgen -G '*')"
+_dvc_cache=""
+_dvc_cache_dir="--global --system --local -u --unset"
+_dvc_checkout="-d --with-deps -R --recursive -f --force $(compgen -G '*.dvc')"
+_dvc_commit="-d --with-deps -R --recursive -f --force $(compgen -G '*.dvc')"
 _dvc_config="-u --unset --local --system --global"
 _dvc_destroy="-f --force"
-_dvc_fetch="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive"
-_dvc_gc="-a --all-branches -T --all-tags -c --cloud -r --remote -f --force -p --project"
-_dvc_import="--resume"
+_dvc_diff="-t --target"
+_dvc_fetch="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive $(compgen -G '*.dvc')"
+_dvc_gc="-a --all-branches -T --all-tags -c --cloud -r --remote -f --force -p --projects -j --jobs"
+_dvc_import="--resume -f --file"
```

Comment:
Looks like you are not using upstream dvc for these commands, right? :slightly_smiling_face: `dvc import` has been changed since 0.41.3. Please install upstream version or at least recently released 0.50.0.

Reply:
I will install the latest version. Thank you